### ISSUE TITLE
download always aborts at ~1GiB

### ISSUE THREAD

**Issue by ThomasWaldmann** [bepasty/bepasty-server#160]
Downloading a binary file with firefox reproducibly aborts at ~1GiB.

Original file uploaded for testing:
```
-rw-r--r-- 1 tw tw 2097152000 Sep 17 09:42  2gb.bin
```

Premature end of download:
```
-rw-r--r-- 1 tw tw          0 Sep 17 12:48 '2gb(1).bin'
-rw------- 1 tw tw 1082366217 Sep 17 12:53 '2gb(1).bin.part'
-rw-r--r-- 1 tw tw          0 Sep 17 13:05 '2gb(2).bin'
-rw------- 1 tw tw 1083705409 Sep 17 13:08 '2gb(2).bin.part'
```

**Comment by ThomasWaldmann** [comment]
```
$ wget https://paste.thinkmo.de/xxxxxxxx/+download
--2018-09-18 13:52:34--  https://paste.thinkmo.de/xxxxxxxx/+download
Resolving paste.thinkmo.de (paste.thinkmo.de)... 213.239.209.163
Connecting to paste.thinkmo.de (paste.thinkmo.de)|213.239.209.163|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 2097152000 (2,0G) [application/octet-stream]
Saving to: ‘+download’

+download            51%[=========>          ]   1,01G  6,38MB/s    in 2m 45s  

2018-09-18 13:55:19 (6,28 MB/s) - Connection closed at byte 1082871971. Retrying.
```
... and the the retry starts from 0 again ...

**Comment by ThomasWaldmann** [comment]
I did a localhost test with the built-in server of werkzeug and downloading 2GiB over http worked.
